### PR TITLE
Added SMP communitymeeting details to table

### LINKED ISSUE #2000: Add SMP Community meeting info to `/community/calendar`
**Current Behavior**

A new SMP meeting has been scheduled on the community calendar.

**Desired Behavior**

The new meeting details should be added to the [/community/calendar](https://layer5.io/community/calendar) page in the table. The details of the meeting are given below:

SMP Community Meeting
Biweekly Thursday
#performance
https://bit.ly/3B7uIy9
https://meet.layer5.io/smp
YouTube playlist to be added

---
**Contributor Resources**

The layer5.io website uses Gatsby, React, and GitHub Pages. Site content is found under the [`master` branch](https://github.com/layer5io/layer5/tree/master).
- See [contributing instructions](https://github.com/layer5io/layer5/blob/master/CONTRIBUTING.md)
- See Layer5 site designs in this [Figma project](https://www.figma.com/file/5ZwEkSJwUPitURD59YHMEN/Layer5-Designs). Fill-in the [Layer5 Community Member Form](https://layer5.io/newcomer) and join the [Layer5 Community Slack](http://slack.layer5.io) for access.

### PR DESCRIPTION
**Pull request by debo19**
Signed-off-by: Debopriya Bhattacharjee <debopriyabh1908@gmail.com>

**Description**

This PR fixes #2000 

**Notes for Reviewers**

**[Signed commits](https://github.com/layer5io/layer5/blob/master/CONTRIBUTING.md#signing-off-on-commits-developer-certificate-of-origin)**
- [x] Yes, I signed my commits.

### CODE REVIEW

**Review by warunicorn19 — APPROVED** [review]
LGTM🎉

**Comment by Tanuj22** [comment]
@debo19 can you revert changes to `package-lock.json`?

**Comment by l5io** [comment]
🚀 Preview for commit b09183e22b67bb7ec482140b22e5de3ef20560da at: https://60f42cbab66d4b8dc603c946--layer5ng.netlify.app

**Comment by debo19** [comment]
Hey @Tanuj22 , I didn't do anything to the package.json  file, I don't know what happened. Can you please tell me how can I revert the changes, I'll do that now

**Comment by Tanuj22** [comment]
> Hey @Tanuj22 , I didn't do anything to the package.json file, I don't know what happened. Can you please tell me how can I revert the changes, I'll do that now

It's probably just due to a difference in node version. Check this out https://dev.to/lofiandcode/git-and-github-how-to-revert-a-single-file-dha . Let me know if you need any more clarification

**Review by navendu-pottekkat — COMMENTED** [review]
Good to merge after reverting the changes in the other file. @adithyaakrishna please update this with the playlist to SMP when you have created it.

Thank you @debo19 🚀

**Comment by debo19** [comment]
I tried to revert the changes. Please let me know if its alright now :)

**Comment by sayantan1413** [comment]
I think its still there because in the `Files Changed` tab there are still 2 files changed so if the revert is successful I think it will show only 1 file changed.

**Comment by debo19** [comment]
@sayantan1413 my second commit shows that the insertions were made. So I'm still unsure if it has happened or not. I think I reverted it twice by mistake.

**Comment by l5io** [comment]
🚀 Preview for commit 0444705747ea6bec010d61a45f21209d0b249a1c at: https://60f46942d262931ae8ea75bb--layer5ng.netlify.app

File: src/sections/Community/Calendar/meetLinksData.js [hunk 1]
```diff
@@ -71,6 +71,15 @@ export const meet_links_data= [
...
     meeting_recordings: "https://www.youtube.com/playlist?list=PL3A-A6hPO2IMPPqVjuzgqNU5xwnFFn3n0",
   },
+  {
+    day: "Biweekly Thursday",
+    meeting: "SMP Community Meeting",
+    slack_channel: "#performance",
+    slack_link: "https://layer5io.slack.com/archives/CVBGPTFJN",
+    meeting_minutes: "https://bit.ly/3B7uIy9",
+    meeting_link: "https://meet.layer5.io/smp",
+    meeting_recordings: "https://www.youtube.com/channel/UCFL1af7_wdnhHXL1InzaMvA",
```
Comment (adithyaakrishna) [inline src/sections/Community/Calendar/meetLinksData.js:81]:
```suggestion
    meeting_recordings: "https://www.youtube.com/playlist?list=PL3A-A6hPO2IPZ3dP6FgZBAA0dZFmmX0kA",
```

**Review by adithyaakrishna — CHANGES_REQUESTED** [review]
@debo19 Could you please make this one change as suggested above? 
Regarding New SMP Community Playlist YouTube

**Comment by debo19** [comment]
@adithyaakrishna Sure I'll add that

**Comment by leecalcote** [comment]
Yes! the youtube playlist channel. :)

**Comment by debo19** [comment]
I don't think these changes to package-lock.json here can be reverted. Should i close this PR and create a new one with youtube playlist added as well?

**Comment by Tanuj22** [comment]
> I don't think these changes to package-lock.json here can be reverted. Should i close this PR and create a new one with youtube playlist added as well?

Sure

**Pull request closed without merging**